decoders/connector/mokosmart/LW006/description.md

```diff
@@ -0,0 +1 @@
+LW006 Smart Badge, the smart badge that combines LoRaWAN network transmission with BLE,WIFI, and GPS positioning technologies. This combination allows for reliable indoor and outdoor positioning, making it ideal for ensuring staff safety, emergency alarm capabilities, and effective personnel management.
```

Comment:
This must be one short sentence quickly stating factual information regarding the sensor. This is one of the fields that will be searched by the Connector page.